src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+
+  let price = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return (price -= LONG_TERM_DISCOUNT);
```

Comment:
You don't need to change `price` variable, as you just return result here
Better to do: `return price - LONG_TERM_DISCOUNT;`